website/docs/d/org_vdc.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_vdc"
+sidebar_current: "docs-vcd-data-source-org-vdc"
+description: |-
+  Provides an organization VDC data source.
+---
+
+# vcd\_org\_vcd
+
+Provides a vCloud Director Organization VDC data source. An Organization VDC can be used to reference a VCD and use its 
+data within other resources or data sources.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc" "my-org-vdc" {
+  org     = "my-org"
+  name    = "my-vdc"
+}
+```
```

Comment:
Please add a few `output` statements to show how to reference the data from this source

Reply:
Added